app/views/courses/index.html.erb

```diff
@@ -1,3 +1,21 @@
-<p>Display all courses for current user.</p>
+<% unless Course.all.empty? %>
```

Comment:
The courses#index is suppose to show all courses that the user is apart of.

Using `Course.all` will look at all courses existing, not just those that relate to the `current_user`.